crates/compiler/builtins/bitcode/src/main.zig

```diff
@@ -238,7 +238,7 @@ pub fn panic(message: []const u8, stacktrace: ?*std.builtin.StackTrace) noreturn
 
 // Run all tests in imported modules
 // https://github.com/ziglang/zig/blob/master/lib/std/std.zig#L94
-test "" {
+test "imported tests" {
```

Comment:
It's now valid to declare an empty test-name like this:
```zig
test {
    // test code
}
```

Reply:
It's actually valid to simply do this in the top-level scope, it won't call it unless running the file as part of a test:
```zig
comptime {
    @import("std").testing.refAllDecls(@This());
}
```

Reply:
Thanks! I'll use the first one, because it's more obvious that it doesn't run unless it's in a test.